compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -482,7 +482,7 @@ def IREEGPU_LaneIdAttr : AttrDef<IREEGPU_Dialect, "LaneId", [
 
 def IREEGPU_ReorderWorkgroupsStrategyAttr :
     EnumAttr<IREEGPU_Dialect, IREEGPU_ReorderWorkgroupsStrategy, ""> {
-  let assemblyFormat = "``$value";
+  let assemblyFormat = "$value";
```

Comment:
I remember this trick from the issue upstream about spaces in gpu attributes but it looks like this is unncessary (none of the ir tests fail?) and also without this, we can't emit bindings for `EnumAttrs` because of [EnumPythonBindingGen.cpp#L154-L165](https://github.com/llvm/llvm-project/blob/d54953ef472bfd8d4b503aae7682aa76c49f8cc0/mlir/tools/mlir-tblgen/EnumPythonBindingGen.cpp#L154-L165).